src/navigation-instruction.js

```diff
@@ -233,16 +233,20 @@ export class NavigationInstruction {
   }
 
   _buildTitle(separator: string = ' | '): string {
-    let title = this.config.navModel.title || '';
+    let title = '';
     let childTitles = [];
 
+    if (this.config.navModel.title) {
+      title = this.router.transformTitle(this.config.navModel.title);
+    }
+
     for (let viewPortName in this.viewPortInstructions) {
       let viewPortInstruction = this.viewPortInstructions[viewPortName];
 
       if (viewPortInstruction.childNavigationInstruction) {
         let childTitle = viewPortInstruction.childNavigationInstruction._buildTitle(separator);
         if (childTitle) {
-          childTitles.push(childTitle);
+          childTitles.push(this.router.transformTitle(childTitle));
```

Comment:
@EisenbergEffect right, good catch. The code has changed since I wrote this, I didn't see this side effect when I updated my fork. I just committed the fix for this. Thx!